include/cinder/Camera.h

```diff
@@ -238,7 +238,8 @@ class CI_API CameraOrtho : public Camera {
 	void	setOrtho( float left, float right, float bottom, float top, float nearPlane, float farPlane );
 
 	bool	isPersp() const override { return false; }
-	
+    //! Returns the four corners of the Camera's Far clipping plane, expressed in world-space
```

Comment:
I'd suggest to remove the description, as to not duplicate the description from the base class. Could you also make sure to fix the white space (I believe Cinder prefers tabs instead of spaces)?